app/src/main/java/com/mashup/mobalmobal/util/NotificationUtils.kt

```diff
@@ -0,0 +1,53 @@
+package com.mashup.mobalmobal.util
+
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import android.os.Build
+import androidx.core.app.NotificationCompat
+import com.mashup.mobalmobal.R
+import com.mashup.mobalmobal.ui.MainActivity
+
+object NotificationUtils {
+    private const val NOTIFICATION_ID = 0
+
+    fun NotificationManager.sendNotification(messageBody: String, applicationContext: Context) {
+
+        val contentIntent = Intent(applicationContext, MainActivity::class.java)
+        contentIntent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP)
+
+        val contentPendingIntent = PendingIntent.getActivity(
+            applicationContext,
+            NOTIFICATION_ID,
+            contentIntent,
+            PendingIntent.FLAG_ONE_SHOT
+        )
+
+        val builder = NotificationCompat.Builder(
+            applicationContext,
+            applicationContext.getString(R.string.notification_channel_id)
+        )
+            .setSmallIcon(R.drawable.common_google_signin_btn_icon_dark)
```

Comment:
```suggestion
        val builder = NotificationCompat.Builder(
            applicationContext,
            applicationContext.getString(R.string.notification_channel_id)
        ) .setSmallIcon(R.drawable.common_google_signin_btn_icon_dark)
```
or
```suggestion
        val builder = NotificationCompat.Builder(
            applicationContext,
            applicationContext.getString(R.string.notification_channel_id)
        ) 
        builder.apply {
            setSmallIcon(R.drawable.common_google_signin_btn_icon_dark)
            setContentTitle~~
            ...
        }
```
이렇게 바꾸는게 예뻐보여요!